packages/components/src/components/assay/AssayResultDeleteModal.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
```

Comment:
nit: It is preferred that we `import React from 'react';` in our new implementations.

Reply:
Ah yes, I fixed this in a couple places but missed this one